src/render_engine/dx12/dx12_render_engine.cpp

```diff
@@ -489,7 +489,7 @@ namespace nova {
                 }
                 texture->SetName(s2ws(texture_name).c_str());
 
-                auto new_tex_index = dynamic_textures.size();
+                const uint32_t new_tex_index = static_cast<uint32_t>(dynamic_textures.size());
```

Comment:
Why not use auto when there's a static cast on the right had side.

Reply:
I don't like auto